examples/plotting/zoom_map_example.py

```diff
@@ -0,0 +1,62 @@
+"""
+===================================
+zoom-out a particular region of map
+===================================
+How to plot a zoom-out image using matplotlib.
+"""
+import matplotlib.pyplot as plt
+from matplotlib.patches import ConnectionPatch
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE
+
+###############################################################################
+# Define a region of interest.
+length = 500 * u.arcsec
```

Comment:
```suggestion
length = 200 * u.arcsec
```

I feel like the original area is too large.

Reply:
yeah!! this is much better. Thank you!